lib/app/App.js

```diff
@@ -53,10 +53,13 @@ export default {
   },
   created () {
     // Add this.$nuxt in child instances
-    Vue.prototype.$nuxt = this
+    Vue.prototype.<%= globals.nuxt %> = this
     // add to window so we can listen when ready
     if (typeof window !== 'undefined') {
-      window.$nuxt = this
+      window.<%= globals.nuxt %> = this
+      <% if (globals.nuxt !== '$nuxt') { %>
+      window.$nuxt = true
```

Comment:
What do you think in apply `pluginPrefix` instead of only `true`?
Will still be a truly and external resources can find the Vue instance (like a vuedevtools).